src/core/Akka.TestKit/TestKitBase_Expect.cs

```diff
@@ -638,7 +628,7 @@ await foreach (var item in enumerable)
         {
             return ExpectMsgAllOfAsync(max, messages, cancellationToken)
                 .ToListAsync(cancellationToken)
-                .ConfigureAwait(false).GetAwaiter().GetResult();
+                .Result;
```

Comment:
We still need to use GetAwaiter().GetResult() because this automatically unwraps all AggregateException, using .Result here breaks behavior consistency.